packages/components/app/styles/components/tabs.scss

```diff
@@ -9,14 +9,14 @@ $indicator-height: 3px;
 .hds-tabs {}
 
 // Sub-components:
-// TODO: MOVE!
+// TODO: MOVE?
```

Comment:
Initially I was thinking child components should live in their own sub-folders with their own styles but maybe not really necessary... Will see.

Reply:
> with their own styles

For context: generally we split the component templates/javascript files, but not the CSS files.